assets/src/js/lazyload.js

```diff
@@ -0,0 +1,16 @@
+export const setupLazyLoad = function() {
```

Comment:
it seems like this is indended with tabs instead of spaces. could you check [eslint](https://circleci.com/gh/greenpeace/planet4-master-theme/5948)?